### ISSUE TITLE
Controlling a computer via Remote does not work

### ISSUE THREAD

**Issue by hdzrvcc0X74** [nvaccess/nvda#18938]
NVDA Remote in the 2026.1 alpha versions does not work as expected.  
A connection can be established between computers running version 2026.1, but control cannot be taken. Pressing any key results in an error.  
The same applies when trying to control a computer running 2026.1 from 2025.3, which is understandable, while controlling a 2025.3 computer from 2026.1 works without problems.  

### Steps to reproduce:

1. Open the "Connect to Another Computer" dialog.
2. Enter the required information.  
3. Once the connection is established, switch to controlling the remote computer.  

### Actual behavior:

NVDA Remote does not work.  

### Expected behavior:

NVDA Remote should work without errors.  

### NVDA logs, crash dumps and other attachments:

ERROR - unhandled exception (20:35:50.160) - MainThread (1616):
Traceback (most recent call last):
  File "wx\core.pyc", line 3427, in <lambda>
  File "_remoteClient\input.pyc", line 231, in sendKey
ctypes.ArgumentError: argument 2: TypeError: expected LP_INPUT instance instead of pointer to INPUT

#### NVDA installed/portable/running from source:

Installed.  

#### NVDA version:

[NVDA alpha-52630,47e6cf5d](https://download.nvaccess.org/snapshots/alpha/nvda_snapshot_alpha-52630,47e6cf5d.exe)  

#### Windows version:

Windows 11 24H2 (AMD64) build 26100.6584  

#### Name and version of other software in use when reproducing the issue:

N/A  

#### Does the issue still occur after restarting your computer?

Yes.  

#### If NVDA add-ons are disabled, is your problem still occurring?

Yes.  

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

Yes.
